unity-renderer/Assets/DCLPlugins/UUIDEventComponentsPlugin/UUIDComponent/OnPointerEvent.cs

```diff
@@ -72,33 +73,42 @@ public class OnPointerEvent : UUIDComponent, IPointerInputEvent, IOutOfSceneBoun
     {
         public static bool enableInteractionHoverFeedback = true;
 
-        [System.Serializable]
+        [Serializable]
         public new class Model : UUIDComponent.Model
         {
             public string button = WebInterface.ACTION_BUTTON.ANY.ToString();
             public string hoverText = "Interact";
             public float distance = 10f;
             public bool showFeedback = true;
 
-            public override BaseModel GetDataFromJSON(string json)
+            public override BaseModel GetDataFromJSON(string json) =>
```

Comment:
Why are we keeping this methods? In the SceneController we removed the old flow messages, this should be no longer needed.

Reply:
I tried to remove it, but it is highly coupled with many Tests. Let's leave it to the next refactoring iteration on sdk6